packages/components/pages/f-checkout/test/component/f-checkout-dinein.component.spec.js

```diff
@@ -5,25 +5,25 @@ const Checkout = require('../../test-utils/component-objects/f-checkout.componen
 let checkout;
 
 describe('f-checkout "dinein" component tests', () => {
-    beforeEach(() => {
+    beforeEach(async () => {
         // Arrange
         checkout = new Checkout();
-        const args = argumentStringBuilder({ serviceType: 'dinein', isLoggedIn: false });
-        checkout.withQuery('args', args);
+        const args = await argumentStringBuilder({ serviceType: 'dinein', isLoggedIn: false });
+        await checkout.withQuery('args', args);
 
         // Act
-        checkout.load();
+        await checkout.load();
     });
 
-    it('should prevent a user from entering more than 12 characters in the tableIdentifier field', () => {
+    it('should prevent a user from entering more than 12 characters in the tableIdentifier field', async () => {
         // Arrange
         const maxlength = 12;
-        const tableEntry = 'A'.repeat(maxlength + 1); // Enter more than allowed
+        const tableEntry = await 'A'.repeat(maxlength + 1); // Enter more than allowed
```

Comment:
We are concatenating value what is the benefit of `Awaiting `it?

Reply:
Same as above.